hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DiskBalancer.java

```diff
@@ -443,7 +444,7 @@ private NodePlan verifyPlanHash(String planID, String plan)
       throws DiskBalancerException {
     final long sha1Length = 40;
     if (plan == null || plan.length() == 0) {
-      LOG.error("Disk Balancer -  Invalid plan.");
+      LOG.error("Disk Balancer -  Invalid plan ().");
```

Comment:
planId ?

Reply:
Thanks @slfan1989 for your comment.
here no need to modify it，i will remove ‘()’